File: backend_mycoffeedau/backend_mycoffeedau/Model/UserVoucher.cs
namespace backend_mycoffeedau.Model
{
    public class UserVoucher
    {
        public int Id { get; set; }
        public string IdUser { get; set; } = string.Empty;
        public string IdVoucher { get; set; } = string.Empty;
        public DateTime RedeemedDate { get; set; } = DateTime.Now;
        public bool IsUsed { get; set; } = false;
    }
}

File: frontend_mycoffeedau/src/models/uservoucher.ts
export interface UserVoucher {
  id: number;
  idUser: string;
  idVoucher: string;
  redeemedDate: string; // ISO string
  isUsed: boolean;
  // Các field join từ backend (GetMyVouchers trả về object join)
  content?: string;
  pointRequirement?: string;
}